plugins/woocommerce-admin/client/core-profiler/services/country.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import { decodeEntities } from '@wordpress/html-entities';
+import { Country } from '@woocommerce/data';
+function decodeHtml( content: string ) {
+	if ( ! content ) {
+		return '';
+	}
+
+	return decodeEntities( content );
+}
+/**
+ * Type definitions
+ */
+
+export type CountryStateOption = {
+	key: string;
+	label: string;
+};
+
+type UserLocation = {
+	latitude: string;
+	longitude: string;
+	country_short: string;
+	country_long: string;
+	region: string;
+	city: string;
+};
+
+/**
+ * Get all country and state combinations used for select dropdowns.
+ *
+ * @return {Object} Select options, { value: 'US:GA', label: 'United States - Georgia' }
+ * @param  countries {Country[]} Countries object.
+ */
+export function getCountryStateOptions(
+	countries: Country[]
+): CountryStateOption[] {
+	const countryStateOptions: CountryStateOption[] = countries.reduce(
+		( acc, country ) => {
+			if ( ! country.states.length ) {
+				acc.push( {
+					key: country.code,
+					label: decodeHtml( country.name ),
+				} );
+				return acc;
+			}
+
+			const countryStates = country.states.map( ( state ) => {
+				return {
+					key: country.code + ':' + state.code,
+					label:
+						decodeHtml( country.name ) +
```

Comment:
I think it's okay to just use `decodeEntities` as we have been using it in current OBW.

https://github.com/woocommerce/woocommerce/blob/b01894f17b5b19154f666afbbc11a9caa6926017/plugins/woocommerce-admin/client/dashboard/components/settings/general/store-address.tsx#L75-L83

Reply:
Thank you for the suggestion 👍 

Updated in https://github.com/woocommerce/woocommerce/pull/38019/commits/d26c0e1a23c48f45934e9205ca051008006fb1b0